### PR TITLE
Added “CocoaPods Prune Localizations”

### PR DESCRIPTION
**Pull request by dtorres**
As requested in #46

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -152,6 +152,14 @@
       "social_media_url":"https://twitter.com/mikejowens",
       "url":"https://github.com/mowens/cocoapods-links",
       "description":"A CocoaPods plugin to manage local development pods"
+    },
+    {
+      "gem": "cocoapods-prune-localizations",
+      "name": "CocoaPods Prune Localizations",
+      "author": "Diego Torres",
+      "social_media_url": "https://twitter.com/dtorres"
```
Comment (segiddins) [inline plugins.json]: this is missing a comma, and thus is invalid JSON

Comment (dtorres) [inline plugins.json]: Sorry about that, fixed.

**Comment by dtorres** [comment]
Added the missing comma

**Comment by orta** [comment]
Looks good :+1:

**Pull request merged**